virtualizarr/manifests/array.py

```diff
@@ -360,7 +360,11 @@ def _possibly_expand_trailing_ellipsis(
     final_dim_indexer = indexer[-1]
     if final_dim_indexer == ...:
         if len(indexer) > ndim:
-            raise ValueError(f"Invalid indexer for array with ndim={ndim}: {indexer}")
+            raise ValueError(
+                f"Indexer `{indexer}` length exceeds number of dimensions `{ndim}`."
```

Comment:
I'm just being extremely nitpicky here but I like it when error messages start with an extremely concise description of what went wrong (and one that doesn't change so is easily greppable), then follow it up with more and more info, something like this:
```suggestion
                f"Invalid indexer for array. Indexer length must be less than or equal to the number of dimensions in the array, but indexer={indexer} has length {len(indexer)} and array has {ndim} dimensions."
```